pkg/templates/charts/toggle/hypershift/templates/hypershift-addon-manager_clusterrole.yaml

```diff
@@ -0,0 +1,112 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: {{ .Values.org }}:{{ .Chart.Name }}:hypershift-addon-manager
+rules:
+- apiGroups:
+  - ""
+  resources:
+  - secrets
+  - configmaps
```

Comment:
@zkayyali812 can you please add `events` over here

Reply:
updated